stats.md

```diff
@@ -4450,25 +4511,25 @@
 
 | | Count | Ratio | 
 |---|---:|---:|
-| Allocations from freelist | 4317382481 | 100.8% |
-| Frees to freelist | 4337069365 |  |
-| Allocations | 4282115839 | 100.0% |
-| Allocations to 512 bytes | 4207486027 | 98.3% |
-| Allocations to 4 kbytes | 59899573 | 1.4% |
-| Allocations over 4 kbytes | 14730239 | 0.3% |
-| Frees | 4332399183 |  |
-| New values | 71765311 |  |
-| Interpreter increfs | 46416805211 | 62.1% |
-| Interpreter decrefs | 56920452833 | 68.7% |
-| Increfs | 28379597077 | 37.9% |
-| Decrefs | 25909369694 | 31.3% |
-| Materialize dict (on request) | 3163292 | 4.4% |
-| Materialize dict (new key) | 1998999 | 2.8% |
+| Allocations from freelist | 4327838130 | 101.9% |
```

Comment:
This number seems off... maybe the total allocation count isn't including free list allocations?

Reply:
Fixed in the latest stats.